src/main/java/io/github/nucleuspowered/nucleus/modules/admin/commands/BlockZapCommand.java

```diff
@@ -39,14 +39,15 @@
 
     @Override public CommandResult executeCommand(CommandSource src, CommandContext args) throws Exception {
         Location<World> location = args.<Location<World>>getOne(this.locationKey).get();
+        String getBlockZapped = location.getBlockType ().getName ();
```

Comment:
Appreciate it! Would it be worth adding the ID as well as the name, do you think?